### ISSUE TITLE
False positives when using let_var_whitespace with string interpolation

### ISSUE THREAD

**Issue by konomae** [realm/SwiftLint#2767]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

False positives when using let_var_whitespace with string interpolation.

```bash
swiftlint version  
0.32.0
```

.swiftlint.yml
```yaml
whitelist_rules:
  - let_var_whitespace
```

main.swift
(maybe, false positive)
```swift
let a = "\(1)"
private let b = "" // Variable Declaration Whitespace Violation: Let and var should be separated from other statements by a blank line. (let_var_whitespace)
```

If I remove string interpolation from the code, no violation found.

```swift
let a = ""
private let b = "" // OK
```

Or removing `private` from second line is also no violation.

```swift
let a = "\(1)"
let b = "" // OK
```

##### Complete output when running SwiftLint, including the stack trace and command used

```bash
$ swiftlint lint
Loading configuration from '.swiftlint.yml'
Linting Swift files at paths 
Linting 'main.swift' (1/1)
/path/to/main.swift:2:1: warning: Variable Declaration Whitespace Violation: Let and var should be separated from other statements by a blank line. (let_var_whitespace)
Done linting! Found 1 violation, 0 serious in 1 file.
```

### Environment

* SwiftLint version (run `swiftlint version` to be sure)? `0.32.0`
* Installation method used (Homebrew, CocoaPods, building from source, etc)? `homebrew`
* Paste your configuration file:

```yml
whitelist_rules:
  - let_var_whitespace
```

**Comment by marcelofabri** [comment]
This is caused by a SourceKit bug: https://github.com/realm/SwiftLint/pull/2720#issuecomment-483112662

**Comment by marcelofabri** [comment]
Looks like this was fixed as part of Swift 5.1 💯
